server/interactors/email.js

```diff
@@ -20,14 +28,21 @@ module.exports.subscribe = function(email, callback) {
         }
     }, function(err, result) {
         var error = null;
-        if (result.error_count > 0) {
+        console.log("err:", err);
```

Comment:
Let's use `console.error` for error logging. Also, this should only be done when err isn't null.